archivist.py

```diff
@@ -3,6 +3,12 @@
 import dotenv
 import fsspec
 import os
+from shutil import rmtree
+
+GREEN = '\033[32m'
+RESET = '\033[0m'
+YELLOW = '\033[93m'
+LIGHTBLUE = '\033[94m'
```

Comment:
Apparently you can use the colorama library to print coloured text while also supporting Windows (Windows doesn't always support ANSI codes). Found out about it here: `https://stackoverflow.com/a/3332860`

Remember to add it to `requirements.txt` as well

Reply:
Totally forgot about Windows :sweat_smile: